src/EnergyPlus/DataSizing.hh

```diff
@@ -339,146 +339,66 @@ namespace DataSizing {
         ZoneSizing zoneSizingMethod = ZoneSizing::Invalid; // load to sizing on: sensible, latent, sensibleandlatent, sensibleonlynolatent
     };
 
-    struct ZoneSizingData
+    // based on ZoneSizingData but only member variables that are actually used by terminal unit sizing
+    struct TermUnitZoneSizingCommonData
```

Comment:
Move common variables in `DataSizing::ZoneSizingData` and `TermUnitZoneSizingData` to a parent struct `TermUnitZoneSizingCommonData`.